drivers/ethernet/eth_enc28j60.c

```diff
@@ -382,18 +382,18 @@ static void eth_enc28j60_init_mac(struct device *dev)
 	}
 
 	/* Configure MAC address */
-	eth_enc28j60_set_bank(dev, ENC28J60_REG_MAADR0);
-	eth_enc28j60_write_reg(dev, ENC28J60_REG_MAADR0,
+	eth_enc28j60_set_bank(dev, ENC28J60_REG_MAADR1);
```

Comment:
The enc28j60 was working mostly ok last time I tried without these changes. What are the symptoms if these changes are not done, could you describe these also in commit msg.

Reply:
It's a mismatch with the latest reference manual, there is no MAADR0 register.

Reply:
Ok. I just have not seen issues regarding Ethernet functionality, so wondering how this mismatch shows itself.